test/commands/extended/get_new_addresses_test.py

```diff
@@ -446,3 +475,20 @@ def test_get_addresses_online(self):
         },
       ],
     )
+
+  def test_new_address_checksum(self):
+    """
+    Generate address with a checksum.
+    """
+    response =\
+      self.command(
+        count         = 1,
+        index         = 0,
+        seed          = self.seed,
+        checksum      = True,
+      )
+
+    self.assertDictEqual(
+      response,
+      {'addresses': [self.addy_1_checksum]},
+    )
```

Comment:
These tests look great, thank you!

Since `AddressGenerator._generate_address()` now also has a `checksum` argument, could you add a test to `AddressGeneratorTestCase` as well?

That way, if we ever modify `GetNewAddressesCommand` and change the way it does checksums, we will still have coverage for `AddressGenerator`'s checksum functionality.